exporter/collector/metricsexporter.go

```diff
@@ -381,7 +363,6 @@ func (m *metricMapper) metricToTimeSeries(
 			ts := m.exponentialHistogramToTimeSeries(resource, extraLabels, metric, eh, points.At(i))
 			timeSeries = append(timeSeries, ts)
 		}
-	// TODO: add cases for other metric data types
 	default:
 		// TODO: log unsupported metric
```

Comment:
This should be a small fix, or should we open a bug for this one?

Reply:
There aren't any other data types, except DataTypeNone (which isn't a real data-type, just the default value for datatype)

Reply:
But more could be added to the protocol ([this oneof](https://github.com/open-telemetry/opentelemetry-proto/blob/c444081593837fe770c825a83c864de736a768b4/opentelemetry/proto/metrics/v1/metrics.proto#L176)) and old collectors should at least warn that they can't be handled. Or does the collector strip these out before they reach the exporter?

Reply:
That would be handled by the default case below, right?

Reply:
Ya, shouldn't we log something though rather than silently dropping?